dev/tasks/webpack.js

```diff
@@ -0,0 +1,57 @@
+/* jshint node: true */
+
+'use strict';
+
+var path = require( 'path' ),
+	MinifyPlugin = require( 'babel-minify-webpack-plugin' );
+
+module.exports = function( grunt ) {
+	grunt.config.merge( {
+		webpack: {
+			all: function() {
+				return {
+					entry: path.join( __dirname, 'utils', 'cs-build-entry.js' ),
+
+					output: {
+						filename: 'cs.js',
+						path: path.join( __dirname, '..', '..', 'plugins', 'easyimage', 'lib' ),
+						libraryTarget: 'var',
+						library: ''
+					},
+
+					module: {
+						rules: [
+							{
+								test: /\.js$/,
+								loader: 'babel-loader',
+								query: {
+									cacheDirectory: true,
+									presets: [
+										[ require( 'babel-preset-env' ), {
+											modules: false,
+											targets: {
+												browsers: [ 'last 2 versions', 'ie >= 8' ]
```

Comment:
We could even further relax this query. We're targeting at Chrome, Firefox, Edge, Safari ATM.